plugins/inputs/tail/tail.go

```diff
@@ -146,7 +146,11 @@ func (t *Tail) receiver(tailer *tail.Tail) {
 
 		m, err = t.parser.ParseLine(text)
 		if err == nil {
-			t.acc.AddFields(m.Name(), m.Fields(), m.Tags(), m.Time())
+			if m != nil {
+				tags := m.Tags()
+				tags["filename"] = tailer.Filename
```

Comment:
Can you change the name of this tag to `path`?  This will just be more consistent with the logparser input, and I hope to combine these plugins at some point.